README.md

```diff
@@ -1 +1,308 @@
 # backend-capstone-template
+# ChatGPT Devs Era
+
+
+ChatGPT Devs Era is a website, customers can browse through various cooks and their menus, discovering a wide range of homemade food options. They can easily place orders for the meals they desire and have them delivered or arrange for pick-up.
+For Cooks, the platform provides a simple way to create profiles and showcase their homemade dishes. They can manage their menu, track orders, and interact with their customers.
+
+## Table of Contents
+
+- [Purpose](#purpose)
+- [Features](#features)
+- [Installation](#installation)
+- [Usage](#usage)
+- [Examples](#examples)
+- [Contributing](#contributing)
+- [License](#license)
```

Comment:
I think in one of the meetings we said we don't need license part, cause we don't have it

Reply:
I think we said first  we don't need to license part then harith explain what is license so  we said we need it